src/llnode.cc

```diff
@@ -11,7 +11,20 @@
 
 namespace llnode {
 
-using namespace lldb;
+using lldb::SBCommandInterpreter;
+using lldb::SBCommandReturnObject;
+using lldb::SBDebugger;
+using lldb::SBError;
+using lldb::SBExpressionOptions;
+using lldb::SBFrame;
+using lldb::SBMemoryRegionInfo;
```

Comment:
This line breaks on lldb 3.8, which is the default lldb on Ubuntu. (Works fine on Mac and later levels of lldb.)
We use `#ifdef LLDB_SBMemoryRegionInfoList_h_` elsewhere.